<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="1" failed="1" skipped="1">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-07-19T14:26:04 IST" name="Suite" finished-at="2024-07-19T14:26:18 IST" duration-ms="13815">
    <groups>
    </groups>
    <test started-at="2024-07-19T14:26:04 IST" name="Test" finished-at="2024-07-19T14:26:18 IST" duration-ms="13815">
      <class name="day46TestNGListeners.OrangeHRM">
        <test-method is-config="true" signature="setup()[pri:0, instance:day46TestNGListeners.OrangeHRM@13c9d689]" started-at="2024-07-19T14:26:04 IST" name="setup" finished-at="2024-07-19T14:26:17 IST" duration-ms="13002" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="testLogo()[pri:1, instance:day46TestNGListeners.OrangeHRM@13c9d689]" started-at="2024-07-19T14:26:17 IST" name="testLogo" finished-at="2024-07-19T14:26:18 IST" duration-ms="64" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testLogo -->
        <test-method signature="testURL()[pri:2, instance:day46TestNGListeners.OrangeHRM@13c9d689]" started-at="2024-07-19T14:26:18 IST" name="testURL" finished-at="2024-07-19T14:26:18 IST" duration-ms="22" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [https:/opensource-demo.orangehrmlive.com/web/index.php/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [https:/opensource-demo.orangehrmlive.com/web/index.php/auth/login] but found [https://opensource-demo.orangehrmlive.com/web/index.php/auth/login]
at org.testng.Assert.fail(Assert.java:111)
at org.testng.Assert.failNotEquals(Assert.java:1578)
at org.testng.Assert.assertEqualsImpl(Assert.java:150)
at org.testng.Assert.assertEquals(Assert.java:132)
at org.testng.Assert.assertEquals(Assert.java:656)
at org.testng.Assert.assertEquals(Assert.java:666)
at day46TestNGListeners.OrangeHRM.testURL(OrangeHRM.java:42)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testURL -->
        <test-method signature="testTitle()[pri:3, instance:day46TestNGListeners.OrangeHRM@13c9d689]" started-at="2024-07-19T14:26:18 IST" name="testTitle" finished-at="2024-07-19T14:26:18 IST" depends-on-methods="day46TestNGListeners.OrangeHRM.testURL" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method day46TestNGListeners.OrangeHRM.testTitle() on instance day46TestNGListeners.OrangeHRM@13c9d689 depends on not successfully finished methods 
[day46TestNGListeners.OrangeHRM.testURL() on instance day46TestNGListeners.OrangeHRM@13c9d689]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method day46TestNGListeners.OrangeHRM.testTitle() on instance day46TestNGListeners.OrangeHRM@13c9d689 depends on not successfully finished methods 
[day46TestNGListeners.OrangeHRM.testURL() on instance day46TestNGListeners.OrangeHRM@13c9d689]
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:159)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- testTitle -->
        <test-method is-config="true" signature="teardown()[pri:0, instance:day46TestNGListeners.OrangeHRM@13c9d689]" started-at="2024-07-19T14:26:18 IST" name="teardown" finished-at="2024-07-19T14:26:18 IST" duration-ms="250" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teardown -->
      </class> <!-- day46TestNGListeners.OrangeHRM -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
